lib/osf-components/addon/components/registries/update-dropdown/template.hbs

```diff
@@ -27,6 +27,7 @@
             {{#if this.revisions}}
                 {{#each this.revisions as |revision index|}}
                     <Registries::UpdateDropdown::ListItem
+                        @onRevisionSelect={{action this.onRevisionSelect dd.close}}
```

Comment:
I tried to do something more simple like `@onClick={{dd.close}}`, but doing so caused the page to do a full load every time. I don't think this is the best way of closing the dropdown on click, but other attempts have either run into the problem where it does a full app load as opposed to a transition, or doesn't show which revision is the currently selected one (like when using `<OsfLink>`

Reply:
Is there any reason why it does a full page load ?